modules/optimization/doc/content/modules/optimization/examples/2d_mbb_pde.md

```diff
@@ -0,0 +1,45 @@
+# 2D MBB Beam with a PDE Filter
```

Comment:
delete this md file and just make a link on the index.md page to the version in the combined/examples.  Also add links in optimization/examples/index.md to the other examples in the combined app that Antonio added like the multi-material and multi-physics examples.  Also add the correct figure to the combined module write-ups that you have here.